doc/api/training/smd_data_parallel.rst

```diff
@@ -70,3 +70,12 @@ model.
 
    smd_data_parallel_pytorch
    smd_data_parallel_tensorflow
+
+Latest Updates
+==============
+
+New features, bug fixes, and improvements are regularly made to the SageMaker distributed data parallel library.
+
+To see the the latest changes made to the library, refer to the library
+`Release Notes
+<https://github.com/aws/sagemaker-python-sdk/blob/master/doc/api/training/smd_data_parallel_release_notes/>`_.
```

Comment:
PR build failing doc check:

```
/codebuild/output/src818993926/src/github.com/aws/sagemaker-python-sdk/doc/api/training/smd_data_parallel.rst:81: D005 No newline at end of file
```

looks like your [editor/ide](https://thoughtbot.com/blog/no-newline-at-end-of-file) is not properly configured to produce posix-compliant files ([every line must end in a newline character](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206)).

Reply:
Done.